Protos/step_by_step.proto

```diff
@@ -96,9 +104,18 @@ message CheckInstructionRequest {
     // Optional. The selected item in the file selector.
     string selected_item = 4;
 }
-
+      
 // CheckInstruction response message.
 message CheckInstructionResponse {
     // Result of the check of the instruction.
-    patternpal.RecognizeResult result = 1;
+    bool result = 1;
```

Comment:
Why are changing this to be a `bool`? In the case of a correct implementation this is fine, but what if the user made a mistake? It would be better to show the results, so they can see what they did incorrectly.

Reply:
I'm having trouble using that RecognizeResult because I do not know what information can be retrieved from the ICheckresult. I have found MatchedNode which is useful and the feedback messages.